tools/cldr-apps/js/src/esm/cldrProgress.js

```diff
@@ -17,6 +17,8 @@ import { createCldrApp } from "../cldrVueRouter";
 import { notification } from "ant-design-vue";
 
 const USE_NEW_PROGRESS_WIDGET = true;
+const CAN_GET_VOTER_PROGRESS = false;
+const CAN_GET_LOCALE_PROGRESS = false;
```

Comment:
why are these constants? I don't see the utility of `CAN_GET_LOCALE_PROGRESS = false;` when it's already hidden if it fails… 

Reply:
that's to avoid needless http requests (for performance) and browser console error messages

also a switch is needed at least for VOTER progress since otherwise the back end does return data but it's inaccurate

Reply:
hm… OK. if that's a problem then i'm ok with this

Reply:
thanks!